opba-protocols/opba-protocol-api/src/main/java/de/adorsys/opba/protocol/api/UpdateAuthorization.java

```diff
@@ -0,0 +1,8 @@
+package de.adorsys.opba.protocol.api;
+
+import de.adorsys.opba.consentapi.model.generated.InlineResponse200;
```

Comment:
The same question as above: if we use it in API, then this API becomes dependency to RestAPI

Reply:
True, atm. it mostly to make it work